lib/evmone/baseline.hpp

```diff
@@ -43,11 +40,8 @@ class CodeAnalysis
         m_padded_code{std::move(padded_code)}
     {}
 
-    CodeAnalysis(bytes_view code, JumpdestMap map, uint8_t version, CodeOffsets offsets)
-      : executable_code{code},
-        jumpdest_map{std::move(map)},
-        eof_version{version},
-        code_offsets{std::move(offsets)}
+    CodeAnalysis(bytes_view code, JumpdestMap map, EOF1Header header)
```

Comment:
```suggestion
    CodeAnalysis(bytes_view code, EOF1Header header)
```
This constructor should not have jumpdest map.